app/engine/products/WalletProduct.ts

```diff
@@ -75,6 +78,20 @@ export class WalletProduct {
             default: null,
             dangerouslyAllowMutability: true
         });
+
+        this.disabledJettons = engine.cloud.get('jettons-disabled', (src) => { src.disabled = {} });
+        if (!storage.getBoolean('disabled-jettons-migration')) {
```

Comment:
Is this based on cloud, or every install it would run migration?

Reply:
I will only go through prevDisabled adding keys to cloud value, but on re-install prevDisabled.length is zero so no migration will commence 'storage' is not 'storagePersistence' and is cleared only in the event of Loggin out or Deleting an account.